source/tests/system/nidigital_driver_api_tests.cpp

```diff
@@ -327,6 +327,53 @@ TEST_F(NiDigitalDriverApiTest, SelfCalibrate_CompletesSuccessfully)
   expect_api_success(response.status());
 }
 
+TEST_F(NiDigitalDriverApiTest, ClockGeneratorInitiate_ClockGenerationInitiated)
+{
+  std::string channel_list = "0";
+
+  ::grpc::ClientContext context;
+  digital::ClockGeneratorInitiateRequest request;
+  request.mutable_vi()->set_id(GetSessionId());
+  request.set_channel_list(channel_list);
+  digital::ClockGeneratorInitiateResponse response;
+  ::grpc::Status status = GetStub()->ClockGeneratorInitiate(&context, request, &response);
+
+  EXPECT_TRUE(status.ok());
+  expect_api_success(response.status());
+  EXPECT_TRUE(get_bool_attribute("0", digital::NIDIGITAL_ATTRIBUTE_CLOCK_GENERATOR_IS_RUNNING));
+}
+
+TEST_F(NiDigitalDriverApiTest, ConfigureSoftwareEdgeStartTrigger_SoftwareEdgeStartTriggerConfigured)
+{
+  ::grpc::ClientContext context;
+  digital::ConfigureSoftwareEdgeStartTriggerRequest request;
+  request.mutable_vi()->set_id(GetSessionId());
+  digital::ConfigureSoftwareEdgeStartTriggerResponse response;
+  ::grpc::Status status = GetStub()->ConfigureSoftwareEdgeStartTrigger(&context, request, &response);
+
+  EXPECT_TRUE(status.ok());
+  expect_api_success(response.status());
+  int start_trigger_type = get_int32_attribute("", digital::NIDIGITAL_ATTRIBUTE_START_TRIGGER_TYPE);
+  EXPECT_EQ(start_trigger_type, digital::TriggerType::TRIGGER_TYPE_NIDIGITAL_VAL_SOFTWARE);
```

Comment:
First parameter for `EXPECT_EQ` should be expected value. Move the expected attribute trigger type value as first parameter.

Reply:
changed that